build/installEdgeDriver.ps1

```diff
@@ -0,0 +1,19 @@
+$url = "https://msedgedriver.azureedge.net/119.0.2151.72/edgedriver_win64.zip" #Chrome Driver from https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/
+$fileName = "edgedriver_win64.zip"
+$source = "C:\Downloads\$fileName"
+$destination = "C:\Program Files\dotnet\"
```

Comment:
Use the %programfiles% ? https://stackoverflow.com/questions/9594066/how-to-get-program-files-x86-env-variable#:~:text=The%20%25ProgramFiles%25%20variable%20points%20to,%22C%3A%5CProgram%20Files%22.